website/docs/sdk-reference/ruby.md

```diff
@@ -187,6 +196,138 @@ configcat_client.force_refresh();
 value = configcat_client.get_value("key", "my default value") # Returns "value from server"
 ```
 
+## Flag Overrides
+
+With flag overrides you can overwrite the feature flags & settings downloaded from the ConfigCat CDN with local values.
+Moreover, you can specify how the overrides should apply over the downloaded values. The following 3 behaviours are supported:
+
+- **Local/Offline mode** (`OverrideBehaviour.LOCAL_ONLY`): When evaluating values, the SDK will not use feature flags & settings from the ConfigCat CDN, but it will use all feature flags & settings that are loaded from local-override sources.
```

Comment:
```suggestion
- **Local/Offline mode** (`ConfigCat::OverrideBehaviour::LOCAL_ONLY`): When evaluating values, the SDK will not use feature flags & settings from the ConfigCat CDN, but it will use all feature flags & settings that are loaded from local-override sources.
```